plugins.txt

```diff
@@ -152,7 +152,6 @@ workflow-aggregator:596.v8c21c963d92d
 workflow-api:1208.v0cc7c6e0da_9e
 workflow-basic-steps:1010.vf7a_b_98e847c1
 workflow-cps:3641.vf58904a_b_b_5d8
-# TODO https://github.com/jenkinsci/workflow-durable-task-step-plugin/pull/299
```

Comment:
@smerle33 please keep this comment until the corresponding PR is merged.

Need another PR like this one: https://github.com/jenkins-infra/docker-jenkins-weekly/pull/1148